docs/contributing/SELF_REVIEW.md

```diff
@@ -0,0 +1,7 @@
+### Self-Review
+
+Before submitting your Pull Request, please check the following items:
+
+- [ ] Are there any typos?
+- [ ] Are all the links included?
+- [ ] Have you made any unnecessary changes?
```

Comment:
```suggestion
- [ ] Have you made no unnecessary changes?
```

Reply:
Fixed in the following commit:
https://github.com/yumemi-inc/flutter-yumemi-lints/pull/40/commits/da1c108acf0523ee4b0970fcbfa5e1737972267c